addons/binding/org.openhab.binding.homepilot/src/main/java/org/openhab/binding/homepilot/handler/HomePilotBridgeHandler.java

```diff
@@ -0,0 +1,137 @@
+package org.openhab.binding.homepilot.handler;
```

Comment:
Missing copyright header, should be fixed with the `mvn license:format` I mentioned earlier.